projects/plugins/jetpack/extensions/shared/memberships.js

```diff
@@ -98,3 +102,19 @@ export const reloadPageWithPremiumContentQueryString = function (
 	}
 	document.location.href = newQueryString;
 };
+
+export const getTokenFromCookie = function () {
```

Comment:
This already exist in `donations/view.js`.
Please remove it from there... and import this one to prevent duplication...

Reply:
We had a call about this today and it seems this code is being added to #33800 . I think either approach will work, and looking at your approach I think it might be a little more performant than mine. I'll refactor accordingly.